### ISSUE TITLE
Mandatory columns

### ISSUE THREAD

**Issue by tombaker** [dcmi/dcap#67]
We have so far identified two columns that can "stand alone" in a CSV row:
* Property ID, as the basis of a minimal profile
* Entity Shape ID which, when listed on a row of its own, will be applied to subsequent statements until a new Entity Shape ID is encountered.

I would like to suggest a third candidate for standing alone: annotations. I have already suggested one possible use for this: [pointing to prefix definitions](https://github.com/dcmi/dcap/issues/66#issuecomment-661649402).

**Comment by tombaker** [comment]
In the following minimal CSV, for example, the annotation column could hold an arbitrary amount of text:

prop_id     | annot |
-- | -- |
 dc:title | Namespace prefixes are defined in prefix.cc |

**Comment by kcoyle** [comment]
@tombaker Your example doesn't seem to illustrate "standing alone in a row". Other than that, as there is no limitation on the annotations the namespace information could be included there, but it wouldn't be actionable from there. However, it could it be used to link to a local file by giving the file name?

**Comment by tombaker** [comment]
@kcoyle Good point. Here is the same example but showing the annotation standing alone in a row:

| prop_id     | annot |
| -- | -- |
|| Namespace prefixes are defined in prefix.cc |
| dc:title | |

We have, I think, implicitly agreed that a CSV cannot be said to conform to our model if it does not have at least one property ID.  But we agreed that an entity shape ID _could_ stand alone on a row so, in effect, I am suggesting that annotations also be allowed to stand on their own.

**Comment by kcoyle** [comment]
This implies (or does it?) that an annotation that is on a row with an entity annotates the entity; that an annotation that is on a row with a property annotates the property; and that an annotation that is on a row with neither annotates the profile as a whole. I'm not entirely comfortable with this because it opens up the possibility that admin data would be included in the profile and that somewhat changes the nature of the column.
